src/modules/modal/containers/modal-claim-fees.js

```diff
@@ -169,11 +203,11 @@ const mergeProps = (sP: any, dP: any, oP: any) => {
       ? [
           {
             label: "Total REP",
-            value: `${sP.reportingFees.unclaimedRep.full} REP`
+            value: `${formatRep(unclaimedRep.toNumber()).full} REP`
```

Comment:
FYI, format-number can take number, string or BigNumber no need to call `.toNumber()`.